dockers/docker-ptf/Dockerfile.j2

```diff
@@ -139,6 +139,28 @@ RUN ln -s /usr/bin/tcpdump /usr/sbin/tcpdump
 
 RUN mkdir -p /var/log/supervisor
 
+# add dependencies to build gnmi_get
+RUN export VERSION=1.14.2 \
+{% if CONFIGURED_ARCH == "armhf" %}
+ && wget https://storage.googleapis.com/golang/go$VERSION.linux-armv6l.tar.gz \
+ && tar -C /usr/local -xzf go$VERSION.linux-armv6l.tar.gz \
+{% elif CONFIGURED_ARCH == "arm64" %}
+ && wget https://storage.googleapis.com/golang/go$VERSION.linux-arm64.tar.gz \
+ && tar -C /usr/local -xzf go$VERSION.linux-arm64.tar.gz \
+{% else %}
+ && wget https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz \
+ && tar -C /usr/local -xzf go$VERSION.linux-amd64.tar.gz \
+{% endif %}
+ && export PATH=$PATH:/usr/local/go/bin \
+ && mkdir /etc/go \
+ && export GOPATH=/etc/go \
+ && rm go$VERSION.linux-*.tar.gz \
+ && cd /etc/go \
+ && git clone https://github.com/google/gnxi.git \
+ && cd /etc/go/gnxi \
+ && wget https://github.com/lguohan/gnxi/blob/master/gnmi_cli_py/py_gnmicli.py \
```

Comment:
this is python, so you do not need to install go lang.

here is what you should do:
1. git clone https://github.com/lguohan/gnxi/tree/master/gnmi_cli_py and check out specific commit id. d01b36e 
2. follow instructions https://github.com/lguohan/gnxi/tree/master/gnmi_cli_py to install required tools.

Reply:
@lguohan: Do you plan to open a PR to upstream your patch?